app/src/main/java/com/yagni/view/EmployeeGUI.java

```diff
@@ -143,13 +143,11 @@ public void actionPerformed(ActionEvent e) {
 
         if(e.getSource() == inventoryButton){
```

Comment:
.add spaces after you if statement's, before the condition
 -- > if (blah blah) not if(blah)
 won't effect the code but it's the style were going for

Reply:
Changed now.